packages/common-types/src/accountHistory.ts

```diff
@@ -163,42 +162,37 @@ export interface SwapApiEvent {
 }
 
 export interface EarnDepositApiDataEvent {
-  strategyId: StrategyId;
-  positionId: SdkEarnPositionId;
-  asset: {
-    amount: string;
-    address: Address;
-    price?: number;
-  };
+  owner: Address;
```

Comment:
can't we also have the strategyId so we can link to it in the receipt?

Reply:
Same comment about all other events

Reply:
I have asked for this data, do you think it's a priority right now? I think we might need it for the release, but not as an urgent

Reply:
Yes for the release, not for now